optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -24,18 +24,20 @@
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
 
 import torch
-from transformers import AutoModelForSeq2SeqLM, AutoModelForSpeechSeq2Seq, GenerationConfig
+from transformers import AutoModelForSeq2SeqLM, AutoModelForSpeechSeq2Seq, AutoModelForVision2Seq, GenerationConfig
 from transformers.file_utils import add_start_docstrings_to_model_forward
 from transformers.modeling_outputs import BaseModelOutput, Seq2SeqLMOutput
 
 import onnxruntime as ort
 from huggingface_hub import hf_hub_download
+from onnxruntime import InferenceSession
```

Comment:
Can you actually remove `import onnxruntime as ort` and `from onnxruntime import InferenceSession` and just put `import onnxruntime as ort` inside the `TYPE_CHECKING` block? And quote everything in the rest of the code. I think it is used only for type hints.